internal/internal_task_handlers.go

```diff
@@ -396,7 +396,7 @@ func newWorkflowTaskHandler(
 		dataConverter:                  params.DataConverter,
 		contextPropagators:             params.ContextPropagators,
 		tracer:                         params.Tracer,
-		workflowInterceptors:           params.WorkflowInterceptors,
+		workflowInterceptors:           params.WorkflowInterceptorChainFactories,
```

Comment:
what is a Chain factory?

Reply:
It's an option field related to replay. Here is the link. Here it looks like we need a consolidation on the naming. 
https://sourcegraph.com/github.com/uber-go/cadence-client@master/-/blob/internal/worker.go?L200

Reply:
Unfortunately, both WorkerOptions and ReplayOptions are exported. But they have different names. 
https://sourcegraph.com/github.com/shijiesheng/cadence-client@master/-/blob/worker/worker.go#L191:43
https://sourcegraph.com/github.com/shijiesheng/cadence-client@master/-/blob/worker/worker.go#L178:37